### PR TITLE
test divided in some describes

### PR DESCRIPTION
**Pull request by Enriquesuarezgtte**

### CODE REVIEW

File: test/buy-tshirt.spec.ts [hunk 1]
```diff
@@ -11,36 +11,64 @@ import {
...
+      beforeAll(async () => {
+        const emailKey: string = 'aperdomobo@gmail.com';
+        const passwordKey: string = 'WorkshopProtractor';
+        const signInCardStep: SignInCardStepPage = new SignInCardStepPage();
+
+        await signInCardStep.sendEmailandPasswKeys(emailKey, passwordKey);
+        await signInCardStep.signIn();
+      });
+
+      describe('and select default address', () => {
```
Comment (leonleo997) [inline test/buy-tshirt.spec.ts]: are you selecting a default address?

Comment (Enriquesuarezgtte) [reply test/buy-tshirt.spec.ts]: I'm continuing process with default address

Comment (leonleo997) [reply test/buy-tshirt.spec.ts]: you should specify that in the description

Comment (Enriquesuarezgtte) [reply test/buy-tshirt.spec.ts]: Should be the describe title more clear in that way?

Comment (Enriquesuarezgtte) [reply test/buy-tshirt.spec.ts]: > you should specify that in the description

got it!

Comment (leonleo997) [reply test/buy-tshirt.spec.ts]: yes

**Pull request merged**